core/src/main/scala/spire/algebra/Conjugation.scala

```diff
@@ -0,0 +1,27 @@
+package spire
+package algebra
+
+/** Describes a conjugation, which is an antihomomorphism. It obeys:
+  *
+  *   i. conjugate(1) = 1
+  *  ii. conjugate(x + y) = conjugate(x) + conjugate(y)
+  * iii. conjugate(x*y) = conjugate(y)*conjugate(x)
```

Comment:
I'm pretty sure `conjugate(conjugate(x)) = x` right? Can we add that here (we are testing it).

Reply:
Done in the new version.